packages/core/src/common/styles/_variables.scss

```diff
@@ -17,6 +18,7 @@ $intents: (
   "info": $intent-info,
   "warning": $intent-warning,
   "error": $intent-error,
+  "none": $intent-none,
```

Comment:
could you share screenshots how it looks by default with and without this change ? 
I suspect it could affect many places where ui button etc are used without specifying intent, please make sure example app and hf-ui styles are still good after this ?

Reply:
removed $none variable, as i found solution to resolve none intent message position issue

Reply:
![Снимок экрана от 2022-01-10 15-41-29](https://user-images.githubusercontent.com/81973123/148776920-06d24978-2d23-421d-8829-f7d3a94a5294.png)

Reply:
was:
![Снимок экрана от 2022-01-10 12-59-18](https://user-images.githubusercontent.com/81973123/148776962-81a6d8e3-d208-40ce-b43e-a27c4e09e259.png)